plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -678,6 +681,65 @@ nothrow:
         }
     }
 
+    void opCall(ref IncreaseFilePrio data) {
+        import std.file : exists, readText;
+        import std.json : JSONValue, JSONOptions, parseJSON;
+
+        if (conf.metadataPath.length == 0) {
+            return;
+        } else if (!exists(conf.metadataPath)) {
+            logger.error("File: " ~ conf.metadataPath ~ " does not exist").collectException;
+            return;
+        }
+
+        string fileContent;
+        try {
+            fileContent = readText(conf.metadataPath);
+        } catch (Exception e) {
+            logger.error("Unable to read file " ~ conf.metadataPath).collectException;
+            return;
+        }
+
+        JSONValue jContent;
+        try {
+            jContent = parseJSON(fileContent, JSONOptions.doNotEscapeSlashes);
+        } catch (Exception e) {
+            logger.info(e.msg).collectException;
+            logger.error("Failed to parse filecontent of " ~ conf.metadataPath ~ "into JSON")
+                .collectException;
+            return;
+        }
+
+        JSONValue objectData;
+        try {
+            objectData = jContent["file-prio"];
+        } catch (Exception e) {
+            logger.info(e.msg).collectException;
+            logger.error("Object 'file-prio' not found in file " ~ conf.metadataPath)
+                .collectException;
+            return;
+        }
+
+        Path[] prioFiles;
+        try {
+            foreach (prioFileElem; objectData.arrayNoRef) {
+                string strData = prioFileElem.str;
+                prioFiles ~= Path(strData);
+            }
+        } catch (Exception e) {
+            logger.error("'file-prio' JSON object not a valid array in file " ~ conf.metadataPath)
```

Comment:
also print e.msg as info in case the jsonvalue-exception contains some useful information for the user